.github/workflows/docker-images.yaml

```diff
@@ -0,0 +1,73 @@
+# Test our docker images are built with the most recent version
+name: Docker images
+"on":
+  schedule:
+    # run daily 0:00 on main branch
+    - cron: '0 0 * * *'
+  pull_request:
+  push:
+    tags:
+    - '*'
+    branches:
+    - release_test
+    - trigger/package_test
+jobs:
+  docker_tests:
+    name: Docker image ${{ matrix.image }}
+    runs-on: ubuntu-latest
+    services:
+      ts:
+        image: ${{ matrix.image }}
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    env:
+      PGHOST: localhost
+      PGUSER: postgres
+
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [
+            "timescale/timescaledb:latest-pg13",
+            "timescale/timescaledb:latest-pg14",
+            "timescale/timescaledb:latest-pg15",
+            "timescale/timescaledb:latest-pg16",
+          ]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Read versions
+      id: versions
+      run: |
+        # read expected version from version.config
+        # version will only be a proper version in a release branch so we use update_from_version
+        # as fallback for main
+        if grep '^version = [0-9.]\+$' version.config; then
+          version=$(grep '^version = ' version.config | sed -e 's!^version = !!')
+        else
+          version=$(grep '^update_from_version = ' version.config | sed -e 's!^update_from_version = !!')
+        fi
+        echo "version=${version}" >>$GITHUB_OUTPUT
+
+    - name: Check version
+      run: |
+        psql -c "SELECT extname,extversion,version() FROM pg_extension WHERE extname='timescaledb'"
+        installed_version=$(psql -X -t \
+          -c "SELECT extversion FROM pg_extension WHERE extname='timescaledb';" | sed -e 's! !!g')
+        if [ "${{ steps.versions.outputs.version }}" != "$installed_version" ];then
+          false
+        fi
+
+    - name: Smoke test
+      run: |
+        psql -c "CREATE TABELE metrics(time timestamptz, device text, metric text, value float);"
```

Comment:
Have you considered adding a `create_hypertabe` call as well to invoke the extension code?